src/olympia/stats/tests/test_commands.py

```diff
@@ -258,7 +258,7 @@ def test_is_valid_source(self):
         assert not is_valid_source('foob',
                                    fulls=['foo', 'bar'],
                                    prefixes=['baz', 'cruux'])
-        assert is_valid_source('foobaz',
```

Comment:
Is it worth keeping this assert, but changing it to `assert not` to make sure the bug that existed previously is in fact fixed?